docs/src/cloud/index.md

```diff
@@ -0,0 +1,17 @@
+# About LanceDB Cloud
+
+LanceDB Cloud is a SaaS (software-as-a-service) solution that runs serverless in the cloud, clearly separating storage from compute. It's designed to be cost-effective and highly scalable without breaking the bank. LanceDB Cloud is currently in private beta with general availability coming soon, but you can apply for early access with the private beta release by signing up below.
+
+[Try out LanceDB Cloud](https://noteforms.com/forms/lancedb-mailing-list-cloud-kty1o5?notionforms=1&utm_source=notionforms){ .md-button .md-button--primary }
+
+## Architecture
+
+LanceDB Cloud provides the same underlying fast vector store that powers the OSS version, but without the need to maintain your own infrastructure. Because it's serverless, you only pay for the storage you use, and you can scale compute up and down as needed depending on the size of your data and its associated index.
```

Comment:
Correct. Users don't set the scale. The elastic scale is taken care by us.

Reply:
How would you frame it using that snippet @QianZhu ?

Reply:
How about this?

> Because it's serverless, you only pay for the storage you use, and we take care of scaling compute up and down as needed depending on the size of your data and its associated index.